### PR TITLE
add io.beekeeperstudio.Studio

### PR DESCRIPTION
**Pull request by proletarius101**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/beekeeper-studio/beekeeper-studio/issues/22#issuecomment-1356159252
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: beekeeper-studio.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+set -oue pipefail
+
+export FLATPAK_ID="${FLATPAK_ID:-io.beekeeperstudio.Studio}"
+export TMPDIR="${XDG_RUNTIME_DIR}/app/${FLATPAK_ID}"
+
+# Wayland support can be optionally enabled like so:
+# flatpak override --user --env=USE_WAYLAND=1 io.beekeeperstudio.Studio
+declare -i USE_WAYLAND="${USE_WAYLAND:-0}"
```
Comment (hfiguiere) [inline beekeeper-studio.sh:10]: not with the socket permissions.

if upstream doesn't support wayland out of the box, flatpak definitely shouldn't

File: io.beekeeperstudio.Studio.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    <name>Beekeeper Studio</name>
+    <summary>The SQL Editor and Database Manager Of Your Dreams</summary>
+
+    <metadata_license>CC0-1.0</metadata_license>
+    <project_license>GPL-3.0</project_license>
+    <developer_name>Beekeeper Studio, Inc</developer_name>
+
+    <description>
+        <p>
+            <i>NOTE: This wrapper is not verified by, affiliated with, or supported by Beekeeper Studio, Inc.</i>
```
Comment (barthalion) [inline io.beekeeperstudio.Studio.metainfo.xml:14]: Why is this disclaimer needed if it's licensed under GPL3?

File: io.beekeeperstudio.Studio.metainfo.xml [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+    <url type="homepage">https://www.beekeeperstudio.io/</url>
+    <url type="vcs-browser">https://github.com/beekeeper-studio/beekeeper-studio</url>
+    <url type="contact">https://docs.beekeeperstudio.io/docs/contact-support</url>
+    <url type="donation">https://github.com/sponsors/beekeeper-studio</url>
+    <url type="bugtracker">https://github.com/beekeeper-studio/beekeeper-studio/issues</url>
+
+    <launchable type="desktop-id">io.beekeeperstudio.Studio.desktop</launchable>
+    <screenshots>
+        <screenshot type="default">
```
Comment (barthalion) [inline io.beekeeperstudio.Studio.metainfo.xml:48]: There can be only one default screenshot.

File: io.beekeeperstudio.Studio.yml [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+            .assets[] | select(.name == "Beekeeper-Studio-" + $version + "-arm64.AppImage")
+            | .browser_download_url
+
+      - type: file
+        path: beekeeper-studio.sh
+
+      - type: file
+        path: io.beekeeperstudio.Studio.metainfo.xml
+
+    build-commands:
```
Comment (barthalion) [inline io.beekeeperstudio.Studio.yml:52]: It doesn't really matter, but by convention, `sources` are the last element of a module.

File: beekeeper-studio.sh (hunk 1 above)
Comment (barthalion) [reply beekeeper-studio.sh:10]: If Ozone and Wayland are not enabled by upstream by default, I don't think it makes much sense to add it in the script.

File: io.beekeeperstudio.Studio.metainfo.xml (hunk 2 above)
Comment (barthalion) [reply io.beekeeperstudio.Studio.metainfo.xml]: It's convention for proprietary apps we have no redistribution rights for. It's not supposed to be used for free software.

Comment (barthalion) [reply io.beekeeperstudio.Studio.metainfo.xml]: The trademark, if there is one, is covered by nominative fair use.

Comment (barthalion) [reply io.beekeeperstudio.Studio.metainfo.xml]: It's what we've been through already with some vendors requesting removal, yes.

**Pull request closed without merging**